packages/aws-cdk/README.md

```diff
@@ -560,6 +561,125 @@ This feature currently has the following limitations:
+| Provided Template + stack-name is from a deployed stack in the account/region                     | The CDK application will deploy as a changeset to the existing stack          |
+| Provided Template has no overlap with resources already in the account/region                     | The CDK application will deploy a new stack successfully                      |
+| Provided Template has overlap with Cloudformation managed resources already in the account/region | The CDK application will not be deployable unless those resources are removed |
+| Provided Template has overlap with unmanaged resources already in the account/region              | The CDK application will not be deployable until those resources are adopted with CDK Import |
+
+
+##### **The provided template is already deployed to CloudFormation in the account/region**
+
+If the provided template came directly from a deployed CloudFormation stack, and that stack has not experienced any drift, 
+then the generated application will be immediately deployable, and will not cause any changes to the deployed resources.
+Drift might occur if a resource in your template was modified outside of CloudFormation, namely via the AWS Console or AWS CLI.
+
+##### **The provided template is not deployed to CloudFormation in the account/region, and there *is not* overlap with existing resources in the account/region**
+
+If the provided template represents a set of resources that have no overlap with resources already deployed in the account/region, 
+then the generated application will be immediately deployable. This could be because the stack has never been deployed, or
+the application was generated from a stack deployed in another account/region.
+
+In practice this means for any resource in the provided template, for example,
+
+```Json
+    "S3Bucket": {
+      "Type": "AWS::S3::Bucket",
+      "Properties": {
+        "BucketName": "MyBucket",
+        "AccessControl": "PublicRead",
+      },
+      "DeletionPolicy": "Retain"
+    }
+```
+
+There must not exist a resource of that type with the same identifier in the desired region. In this example that identifer 
+would be "MyBucket"
+
+##### **The provided template is not deployed to CloudFormation in the account/region, and there *is* overlap with existing resources in the account/region**
+
+If the provided template represents a set of resources overlap with resources already deployed in the account/region, 
+then the generated application will not be immediately deployable. If those overlapped resources are already managed by 
+another CloudFormation stack in that account/region, then those resources will need to be manually removed from the provided
+template. Otherwise, if the overlapped resources are not managed by another CloudFormation stack, then first run remove those
```

Comment:
Is there some words missing here? I'm not sure what this part is supposed to say.

Reply:
Github is highlighting too many lines. I'm asking about this part "Otherwise, if the overlapped resources are not managed by another CloudFormation stack, then first run remove those"

Reply:
I'm not sure what the change should be here.